eng/pipelines/runtime-extra-platforms-other.yml

```diff
@@ -183,7 +183,7 @@ jobs:
     runtimeFlavor: mono
     platforms:
       - MacCatalyst_x64
-      # - MacCatalyst_arm64 # re-enable it here and remove from runtime-staging after it's stabilized on OSX 11.00 queue
+      - MacCatalyst_arm64
```

Comment:
Do we guard this with `isRollingBuild` as well until we get more devices?

Reply:
Good point, thanks! @steveisok @akoeplinger @premun Do you happen to know whether there is still lack of devices for that?

Reply:
Discussed with @akoeplinger - there is enough device capacity at the moment, so we can try to make both arm64 legs available for manual per-PR run.